src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9037,6 +9037,11 @@ public enum NumberStyles
         AllowBinarySpecifier = 1024,
         BinaryNumber = 1027,
     }
+    public struct NSRange
```

Comment:
Change in this file should be reverted and NSRange change to internal. This is not approved public API.

Reply:
Reverted this change.